src/Tizen.Network.Stc/Tizen.Network.Stc/StcEnumerations.cs

```diff
@@ -0,0 +1,258 @@
+        Wifi,
+        /// <summary>
+        /// Network interface type is  wired.
+        /// </summary>
+        Wired,
+        /// <summary>
+        /// Network interface type is bluetooth.
+        /// </summary>
+        Bluetooth,
+        /// <summary>
+        /// Network interface type is all.
+        /// </summary>
+        All
+    }
+
+    /// <summary>
+    /// Enumeration for network roaming type
+    /// </summary>
+    public enum StcRoamingType
+    {
+        /// <summary>
+        /// Not in roaming.
+        /// </summary>
+        Disabled,
+        /// <summary>
+        /// In roaming.
+        /// </summary>
+        Enabled
+    }
+
+    /// <summary>
+    /// Enumeration for hardware network protocol type. 
+    /// </summary>
+    public enum StcProtocolType
+    {
+        /// <summary>
+        /// Network unknown.
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// Network no service.
+        /// </summary>
+        DatacallNoService,
+        /// <summary>
+        /// Network emergency.
+        /// </summary>
+        DatacallEmergency,
+        /// <summary>
+        /// Network search 1900.
+        /// </summary>
+        DatacallSearch,
+        /// <summary>
+        /// Network 2G.
+        /// </summary>
+        Datacall2G,
+        /// <summary>
+        /// Network 2.5G.
+        /// </summary>
+        Datacall2_5G,
+        /// <summary>
+        /// Network EDGE.
+        /// </summary>
+        Datacall2_5G_Edge,
+        /// <summary>
+        /// Network UMTS.
+        /// </summary>
+        Datacall3G,
+        /// <summary>
+        /// Network HSDPA.
+        /// </summary>
+        DatacallHsdpa,
+        /// <summary>
+        /// Network LTE.
+        /// </summary>
+        DatacallLte
+    }
+
+    /// <summary>
+    /// Enumeration for monitored process state.
+    /// </summary>
+    public enum StcProcessState
+    {
+        /// <summary>
+        /// Unknown state.
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// Foreground state.
+        /// </summary>
+        Foreground,
+        /// <summary>
+        /// Background state.
+        /// </summary>
+        Background
+   }
+
+    /// <summary>
+    /// Enumeration for network connection period type.
+    /// @details Last received/sent mean counting data from the first connection of each interface.
+    /// </summary>
+    public enum StcConnPeriodType
+    {
+        /// <summary>
+        /// Unknown period.
+        /// </summary>
+        Unknown,
+        /// <summary>
+        /// Last received data.
+        /// </summary>
+        LastReceivedData,
+        /// <summary>
+        /// Last sent data.
+        /// </summary>
+        LastSentData,
+        /// <summary>
+        /// Total received data.
+        /// </summary>
+        TotalReceivedData,
+        /// <summary>
+        /// Total sent data.
+        /// </summary>
+        TotalSentData
```

Comment:
Why these are a period?

LastDataReceivedData, LastSentData, TotalReceivedData, TotalSentData

Reply:
removed this whole enum as not used.